.github/workflows/mako.yml

```
@@ -0,0 +1,37 @@
+name: mako-testsuite
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+jobs:
+    mako-testsuite-1_1_4:
+    runs-on: ubuntu-latest
+    env:
+      DD_TESTING_RAISE: true
+    defaults:
+      run:
+        working-directory: mako
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ddtrace
+      - uses: actions/checkout@v2
+        with:
+          repository: sqlalchemy/mako
+          ref: 1.1.4
+          path: mako
+      - uses: actions/setup-python@v2
+        with:
+          python-version: '3.8'
+      - name: Install tox
+        run: pip install tox
+      - name: Create tox env
+        run: tox -e py38 --notest
+      - name: Inject ddtrace
+        run: .tox/py38/bin/pip install ../ddtrace
+      - name: Add pytest configuration for ddtrace
+        run: echo -e "[pytest]\nddtrace-patch-all = 1" > pytest.ini
+      - name: Run tests
+        # test_exception_propagation is broken upstream
+        run: tox -e py38 -- -k 'not test_exception_propagation and not test_memory_consumption'
```

```suggestion
        run: tox -e py
```

Change to "py" env, and we can get rid of the ignores that were copied from the Flask one.